odoo/tools/misc.py

```diff
@@ -1572,13 +1564,7 @@ def format_duration(value):
         return '-%02d:%02d' % (hours, minutes)
     return '%02d:%02d' % (hours, minutes)
 
-
-def _consteq(str1, str2):
-    """ Constant-time string comparison. Suitable to compare bytestrings of fixed,
-        known length only, because length difference is optimized. """
-    return len(str1) == len(str2) and sum(ord(x)^ord(y) for x, y in zip(str1, str2)) == 0
-
-consteq = getattr(passlib.utils, 'consteq', _consteq)
+consteq = hmac_lib.compare_digest
```

Comment:
Deprecate as well? But it's used ~25 times. Maybe put ignore and consteq in separate PRs which fix their callsites?

Reply:
Also if deprecating, suggest `secrets.compare_digest`, it's the exact same function but avoids flagging about `hmac`.

Reply:
Maybe don't create another PR but just deprecate both ignore and consteq (+ fix their call sites) in dedicated commits in this PR